misc/fpm-pack.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+# This script packages rpm, deb, and pacman packages of mgmt with fpm. The
+# first argument is the package type, and all subsequent arguments are the
+# dependencies. Example usage: `./fpm-pack.sh deb dependency1 dependency2`
+
+# the binary to package
+BINARY="mgmt"
+# git tag pointing to the current commit
+TAG=$(git tag -l --points-at HEAD)
+# maintainer email
+MAINTAINER="mgmt@noreply.github.com"
+# project url
+URL="https://github.com/purpleidea/mgmt"
+# project description
+DESCRIPTION="Next generation distributed, event-driven, parallel config management!"
+# project license
+LICENSE="GPLv3"
+# location to install the binary
+PREFIX="/usr/bin"
```

Comment:
Is this correct or should it be `/usr/local/bin/` ?

Reply:
Hmm... Interesting... I guess I always thought custom stuff like fpm packages (not stock distro packages) belonged in `local` or `/opt`. I'm okay leaving it as is, and if someone wants to change this later we can do it then.